assets/src/edit-story/components/canvas/canvasLayout.js

```diff
@@ -67,18 +67,20 @@ function CanvasLayout() {
   const layoutParamsCss = useLayoutParamsCssVars();
 
   return (
-    <Background ref={setBackgroundRef} style={layoutParamsCss}>
-      <CanvasUploadDropTarget>
-        <CanvasElementDropzone>
-          <SelectionCanvas>
-            <DisplayLayer />
-            <FramesLayer />
-            <NavLayer />
-          </SelectionCanvas>
-          <EditLayer />
-        </CanvasElementDropzone>
-      </CanvasUploadDropTarget>
-    </Background>
+    <StyleSheetManager stylisPlugins={[]}>
```

Comment:
But... how does this even work? Why would this fix it? I don't understand!

Reply:
Something about disabling the default `stylis` plugins, that normally do some RTL magic? Anyway, neat that it works, but it should probably come with a comment, as this is _god-tier_ bug fixing ;)

Reply:
@barklund Done, PTAL